Userland/Applications/PixelPaint/ZoomTool.h

```diff
@@ -17,12 +17,12 @@ class ZoomTool final : public Tool {
     ZoomTool();
     virtual ~ZoomTool() override;
 
-    virtual void on_mousedown(Layer&, GUI::MouseEvent& layer_event, GUI::MouseEvent& image_event) override;
+    virtual void on_mousedown(Layer&, MouseEvent&) override;
     virtual GUI::Widget* get_properties_widget() override;
 
 private:
     RefPtr<GUI::Widget> m_properties_widget;
-    double m_sensitivity { 0.1 };
+    double m_sensitivity { 0.5 };
```

Comment:
this change seems unrelated

Reply:
This should belong to the previous commit.